radio/src/targets/horus/board.cpp

```diff
@@ -89,11 +87,19 @@ void interrupt1ms()
   DEBUG_TIMER_START(debugTimerRotEnc);
   checkRotaryEncoder();
   DEBUG_TIMER_STOP(debugTimerRotEnc);
+
+#if defined(LUA)
+  if (telemetryProtocol == PROTOCOL_FRSKY_SPORT) {
+    if (telemetryFifo.last(-2) == 0x7E && telemetryFifo.last(-1) == outputTelemetryBufferTrigger) {
+      sportSendBuffer(outputTelemetryBuffer, outputTelemetryBufferSize);
```

Comment:
Are we sure we still receive telemetry? I remember having this bug, but the RX was completely silent because of a bug inside the S6R